### ISSUE TITLE
Sound doesn't play

### ISSUE THREAD

**Issue by BryceAG** [phetsims/plinko-probability#94]
Test device:
Schrodinger

Operating System:
Windows 8

Browser:
IE 11, Firefox

Problem description:
sound does not play for either the intro or lab section

Steps to reproduce:
load the sim on IE11 or firefox for windows 8, click enable sound. It plays sound correctly for chrome, so the speakers do work.

Severity:

Screenshots:

Troubleshooting information (do not edit):
Name: ‪Plinko Probability‬
URL: http://www.colorado.edu/physics/phet/dev/html/plinko-probability/1.0.0-rc.1/plinko-probability_en.html
Version: 1.0.0-rc.1 2016-09-16 19:30:26 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 6.3; WOW64; rv:49.0) Gecko/20100101 Firefox/49.0
Language: en-US
Window: 1536x755
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 8192 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16383x16383
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"7d27130a","branch":"HEAD"},"axon":{"sha":"e0192608","branch":"HEAD"},"babel":{"sha":"c041de05","branch":"master"},"brand":{"sha":"f0b1f7da","branch":"HEAD"},"chipper":{"sha":"07058555","branch":"HEAD"},"dot":{"sha":"39436598","branch":"HEAD"},"joist":{"sha":"b513b5ee","branch":"HEAD"},"kite":{"sha":"73302899","branch":"HEAD"},"phet-core":{"sha":"c48bf320","branch":"HEAD"},"phetcommon":{"sha":"83ea84c8","branch":"HEAD"},"plinko-probability":{"sha":"3a105d26","branch":"HEAD"},"query-string-machine":{"sha":"05231e54","branch":"HEAD"},"scenery":{"sha":"e78ee413","branch":"HEAD"},"scenery-phet":{"sha":"f0fc9ae8","branch":"HEAD"},"sherpa":{"sha":"bcc28cd6","branch":"HEAD"},"sun":{"sha":"f2bd9d60","branch":"HEAD"},"tandem":{"sha":"d369b847","branch":"HEAD"},"vibe":{"sha":"b422db9c","branch":"HEAD"}}

related to phetsims/tasks#702

**Comment by pixelzoom** [comment]
@BryceAG Do you know if this problem also occurs on Windows 7 with IE11 and FF?

**Comment by phet-steele** [comment]
Closing, audio actually works. See below for details. Pinging @jessegreenberg on Skype for his thoughts.

[3:11:43 PM] Steele Dalton: The audio actually works in all 6 browsers (Win 7 & 8)
[3:12:29 PM] Steele Dalton: I had to go into the task manager on the Win 8 machine and end something called the "Windows Audio Device Graph Isolator", or something close to that name, then everything worked
[3:12:40 PM] Steele Dalton: some trial and error to find that out
[3:12:41 PM] Chris Malley: I see.
[3:12:44 PM] Steele Dalton: BUT
[3:13:02 PM] Steele Dalton: IE11 does play audio at a much slower frequency
[3:13:13 PM] Steele Dalton: not a huge deal though
[3:13:29 PM] Chris Malley: Do you really mean frequency, or do you mean duration?
[3:13:48 PM] Steele Dalton: I mean the tempo, bpm, rate at which I heard beeps
[3:14:09 PM] Steele Dalton: like the balls were dispensing slower (but they weren't)
[3:14:43 PM] Chris Malley: This task manager setting you had to tweak... Does that affect audio for all phet sims?
[3:15:27 PM] Chris Malley: oh... you said you had to kill a task.
[3:15:29 PM] Steele Dalton: no freaking idea man, I've never seen it before and it was a shot in the dark. My logic was literally "go through the task manager and end everything with the word 'audio' in it"
[3:16:00 PM] Chris Malley: is it a task that reappears whenever you restart Windows? and does it appear on machines other than the test machine in question?
[3:16:40 PM] Chris Malley: since plinko is doing some custom audio stuff, please test to see if it affects other sims.
[3:17:49 PM] Steele Dalton: it did restart a few seconds after I killed it, but it did the trick. Prior to killing it, I was getting a message saying that the audio was dedicated to another process and to end that before being able to use the speakers for anything else. That computer has had a bunch of a11y work done to it, so it's probably something there.
[3:18:32 PM] Chris Malley: ah. could be the screen reader (or something a11y) hogging the audio device.
[3:18:40 PM] Chris Malley: check with Jesse?
[3:20:21 PM] Steele Dalton: I'll do what I can, regardless I believe 94 can be closed and we can open a low priority issue about the slower IE11 audio

**Comment by jessegreenberg** [comment]
> That computer has had a bunch of a11y work done to it, so it's probably something there.

@phet-steele I am not aware of extensive accessibility work on that device, what a11y work was done?  If it is just the installed screen readers, they shouldn't be interfering with anything unless explicitly turned on.  In that case you would have heard it talking to you while testing.

**Comment by phet-steele** [comment]
Had a Skype chat, @jessegreenberg will check out the device.

**Comment by jessegreenberg** [comment]
In chat, it sounds like ZoomText might have been on in a background process.  I am not familiar with this AT, but it does have a screen reader in addition to a magnifier, so it might have been hogging audio.  I will try to reproduce with readers turned on, but if I can't I'll close the issue.

**Comment by pixelzoom** [comment]
Not a prerequisite for 1.0.0 publication, marking as deferred.

**Comment by jessegreenberg** [comment]
Testing on Schrodinger 9/28/15.  I cannot fully test with ZoomText turned on, the free trial has expired.  It is likely that it has been expired since this issue was opened.  Even though I cannot open the ZoomText app, some associated support module ( ZoomText UAC Support Module) is running in the background, possibly what @phet-steele noticed.  When I tried to open ZoomText, several warning dialogs appeared, warning of incompatibility with Firefox.  They weren't kidding, Firefox is now completely broken, **no** page loads.  After closing ZoomText, Firefox works again with sound.

Sound works great in IE 11, with or without ZoomText turned on.
Sound works with both NVDA and JAWS turned on as well.

I can see 'Windows Audio Device Graph Isolation' in background processes. Interesting information on that:
https://technet.microsoft.com/en-us/magazine/2009.06.windowsconfidential
Maybe ZoomText caused this to hang, but not really sure and it doesn't seem worth investigating further.

**TL;DR -  Maybe ZoomText caused sound to crash, it is hard to know for certain.  I wasn't able to reproduce this specific issue with any AT turned on.  We learned that ZoomText is completely incompatible with Firefox.**

Closing.
